cumulus/client/service/src/lib.rs

```diff
@@ -572,3 +603,133 @@ where
 
 	Err("Stopping following imported blocks. Could not determine parachain target block".into())
 }
+
+async fn parachain_informant<Block: BlockT, Client>(
+	relay_chain_interface: impl RelayChainInterface + Clone,
+	sync_service: Arc<dyn SyncOracle + Send + Sync>,
+	client: Arc<Client>,
+	para_id: ParaId,
+	metrics: Option<ParachainInformantMetrics>,
+) where
+	Client: HeaderBackend<Block> + Send + Sync + 'static,
+{
+	// Backed blocks.
+	let pending_candidates =
+		match pending_candidates(relay_chain_interface.clone(), para_id, sync_service).await {
```

Comment:
since we're looking at candidates pending availability, we could output them multiple times if they're not included right at the next block.
I think we should be instead looking at CandidateBacked events

Reply:
That's the best way that I'm aware of. This way, we can also log when a candidate times out during availability

Reply:
Yeah good suggestion, in pov-recovery it was actually by design that we take a look at the pending blocks at every block. But here it makes sense to use the events.